src/Tutorial/PatientBrowser/index.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>FHIR Patient browser sample app</title>
+    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.0/js/msal.js"></script>
+    <style>
+    
+    @font-face {
+  font-family: 'Segoe UI';
+  src: url('https://static2.sharepointonline.com/files/fabric/assets/fonts/segoeui-westeuropean/segoeui-semibold.woff2') format('woff2'), url('https://static2.sharepointonline.com/files/fabric/assets/fonts/segoeui-westeuropean/segoeui-semibold.woff') format('woff');
+  font-weight: 600;
+  font-style: normal;
+}
+
+        div {
+            font-family: 'Segoe UI';
+        }
+
+        .options {
+            visibility: hidden;
+        }
+
+
+        #WelcomeMessage {
+            font-size: xx-large;
+        }
+        #myTable {
+            font-family: 'Segoe UI';
+            border-collapse: collapse;
+            width: 60%;
+            text-align: center;
+        }
+        
+        #myTable th {
+            background-color: aliceblue;
+            padding: 8px;
+            padding-top: 12px;
+            padding-bottom: 12px;
+        }
+        
+        #myTable tr:nth-child(even) {
+            background-color: #f1f2f5d0;
+        }
+        #myTable tr:hover {
+            background-color: rgb(216, 224, 252);
+        }
+
+    </style>
+</head>
+
+
+    <div class="container">
+        <div class="leftContainer">
+            <p id="WelcomeMessage">Welcome to the FHIR Patient Browsing Sample Application</p>
+            <button id="SignIn" onclick="signIn()">Sign In</button>
+        </div>
+
+        <div class="options">
+            <div><label for="Please select Synthea data file:"></label></div>
+            <div><input type="file" name="image_file" id="image_file" oninput="fileSelected()" /></div>
+            <div>
+                <!-- <input type="button" value="Load Patients" onclick="loadPatients()" /> -->
+            </div>
+            
+        </div>
+
+        <div class="patientTable">
+                <table id="myTable">
+                        <tr>
+                          <th>Last Name</th>
+                          <th>First Name</th>
+                          <th>Sex</th>
+                          <th>Birth Date</th>
+                          <th>Details</th>
+                        </tr>
+                      </table>
+
+                      
+        </div>
+       
+        <div class="rightContainer">
+            <pre id="json"></pre>
+        </div>
+    </div>
+
+<script>
+    var chooseFileButtonClicked = false;
+    var startidx = 0;
+    var msalConfig = {
+        auth: {
+            clientId: '93498c57-9405-442d-bd5e-04afd4687ed0',
```

Comment:
We should not have references to actual AAD instances and client ids in the code. These are configuration settings and should be replaced with place-holders.